scripts/setup-termux.sh

```diff
@@ -5,10 +5,14 @@ PACKAGES=""
 PACKAGES+=" clang"		# Required for termux-elf-cleaner and C/C++ packages.
 PACKAGES+=" file"		# Used in termux_step_massage().
 PACKAGES+=" gnupg"		# Used in termux_get_repo_files() and build-package.sh.
+PACKAGES+=" lrzip"
```

Comment:
Please add a comment about this package to align with others. The same goes for `lzop` and `lz4` below.

Reply:
Resolved in latest commit